tls/s2n_resume.h

```diff
@@ -67,6 +80,11 @@ typedef enum {
     S2N_STATE_WITH_SESSION_TICKET
 } s2n_client_tls_session_state_format;
 
+typedef enum {
+    S2N_TLS12_SERIALIZED_FORMAT_VERSION = 1,
+    S2N_TLS13_SERIALIZED_FORMAT_VERSION
+} s2n_serial_format_version;
```

Comment:
Where are we using `s2n_serial_format_version`?

Reply:
It's just used for writing and reading the format version in the serialize and deserialize methods. I did change it to an enum though.

Reply:
I see, do we anticipate to use this enum in future PRs?  If yes, then it makes sense to keep it as an enum over macros.

Reply:
There's no reason to make it a macro. An enum is an improvement.